src/components/ExpandableBlock.vue

```diff
@@ -8,27 +8,23 @@
       :src="iconDrawer"
       class="arrow"
     >
-    <div
-      class="title"
-      v-html="title"
-    />
+    <div class="title">
+      <slot name="title" />
+    </div>
     <div
       v-if="expanded"
       class="body"
-      v-html="body"
-    />
+    >
+      <slot name="body" />
```

Comment:
trick: if you do not specify a `name` but simply use `<slot>` then it will render the content block that is not wrapped with `<template slot=..>`

```
<expandable-block>
  <template slot="title">
    HOW TO USE SUPERHERO
  </template>
  <p><b>Download and install the Superhero extension for your browser.</b></p><p /><p>
...
```
instead of

```
  <template slot="title">
    HOW TO USE SUPERHERO
  </template>
  <template slot="body">
    <p><b>Download and install the Superhero extension for your browser.</b></p><p /><p>
...
  </template>
```
it will save a lot of `<template>` tags and reduce clutter

Reply:
Above could be further cleaned up if you replace the title slot with a component property